src/pyaro_readers/eeareader/EEATimeseriesReader.py

```diff
@@ -242,15 +243,15 @@ def _read_metadata(self, folder: str) -> dict:
         filename = Path(folder) / "metadata.csv"
         if not filename.exists():
             raise FileExistsError(f"Metadata file could not be found in {folder}")
-        with open(filename, "r") as f:
+        with filename.open("r") as f:
```

Comment:
why read `metadata.csv` by hand, when you are using `polars` to read the obs files. Can not use the same tool for both?

Reply:
This is the old reader (should be removed?)